package.json

```
@@ -134,6 +134,11 @@
             "Don't automatically launch the app after starting"
           ]
         },
+        "shiny.timeout": {
```

I think it's worth being overly specific here about which timeout we're talking about – there's a good chance we'd deprecate or remove this option if we improve the app launch approach.

Maybe

```suggestion
        "shiny.timeoutOpenBrowser": {
```